Ontology/Asset/Component/HVACCoolingMethod.json

```diff
@@ -0,0 +1,78 @@
+{
+  "@id": "dtmi:digitaltwins:rec_3_3:asset:HVACCoolingMethod;1",
+  "@type": "Interface",
+  "displayName": {
+    "en": "Cooling Method"
+  },
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "type",
+      "displayName": {
+        "en": "Type"
+      },
+      "writable": true,
+      "schema": {
+        "@type": "Enum",
+        "valueSchema": "string",
+        "enumValues": [
+          {
+            "name": "DX",
+            "displayName": {
+              "en": "DX"
+            },
+            "enumValue": "DX"
+          },
+          {
+            "name": "ChilledWater",
+            "displayName": {
+              "en": "Chilled Water"
+            },
+            "enumValue": "chilledWater"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Property",
+      "name": "rows",
+      "displayName": {
+        "en": "Rows"
+      },
+      "writable": true,
+      "schema": "integer",
+      "comment": "Coil row configuration"
+    },
+    {
+      "@type": "Property",
+      "name": "finsPerInch",
```

Comment:
An imperial-specific measurement isn't sufficiently general (and cannot be supported in the OWL version of the ontology, which is metric-only). Could some other property name (and comment) be found? E.g., "finDensity" or something of that nature, with a comment clarifying that this can be expressed in fins per inch OR fins per decimeter, depending on deployment context?

Reply:
Agree with `finDensity` and a comment defining its intended "units" for now. We'll have to clarify the best way to handle these "density" cases where there isn't an easily defined Semantic Unit to define things-per-length/area/volume.